### PR TITLE
Changes to EDT link

### PR DESCRIPTION
**Pull request by SabrinaDfE**
EDT link was broken, replaced with correct link

### CODE REVIEW

**Comment by SabrinaDfE** [comment]
https://ukgovernmentdfe.slack.com/files/UM4N38EQM/F05M8TUTF7U/screenshot_2023-08-10_at_15.52.05.png

**Comment by SabrinaDfE** [comment]
https://www.edt.org/united-kingdom/programmes/national-professional-qualifications-npqs/

File: app/views/early-years-leadership.html [hunk 1]
```diff
@@ -166,7 +166,7 @@ <h2 class="govuk-heading-l">Providers</h2>
 
       <ul class="govuk-list govuk-list--bullet">
         <li><a href="https://www.ambition.org.uk/programmes/npqeyl/">Ambition Institute</a></li>
-        <li><a href="https://www.educationdevelopmenttrust.com/EducationDevelopmentTrust/files/8a/8a14e343-56cf-4c28-8332-3e0fe3177abd.pdf">Education Development Trust</a></li>
+        <li><a href="https://www.edt.org/united-kingdom/programmes/national-professional-qualifications-npqs/">Education Development Trust</a></li>
```
Comment (dalcJ) [inline app/views/early-years-leadership.html:169]:
```suggestion
        <li><a href="https://www.edt.org/united-kingdom/discover-our-npqs/early-years-leadership-npqeyl/">Education Development Trust</a></li>
```

Comment (dalcJ) [reply app/views/early-years-leadership.html:169]: Should this go to the early years specific page?

File: app/views/executive-leadership.html [hunk 2]
```diff
@@ -155,7 +155,7 @@ <h2 class="govuk-heading-l">Providers</h2>
         <li><a href="https://www.ambition.org.uk/programmes/npqel/">Ambition Institute</a></li>
         <li><a href="https://www.outstandingleaders.org/npqel">Best Practice Network (home of Outstanding Leaders Partnership)</a></li>
         <li><a href="https://www.cefel.org.uk/npqel/">Church of England</a></li>
-        <li><a href="https://www.educationdevelopmenttrust.com/our-expertise/our-work-in-the-uk/our-programmes-in-the-uk/national-professional-qualifications-(1)/overview-of-our-npqs">Education Development Trust</a></li>
+        <li><a href="https://www.edt.org/united-kingdom/programmes/national-professional-qualifications-npqs/">Education Development Trust</a></li>
```
Comment (dalcJ) [inline app/views/executive-leadership.html:158]:
```suggestion
        <li><a href="https://www.edt.org/united-kingdom/discover-our-npqs/executive-leadership-npqel/">Education Development Trust</a></li>
```

File: app/views/headship.html [hunk 3]
```diff
@@ -158,7 +158,7 @@ <h2 class="govuk-heading-l">Providers</h2>
         <li><a href="https://www.ambition.org.uk/programmes/npqh/">Ambition Institute</a></li>
         <li><a href="https://www.outstandingleaders.org/npqh">Best Practice Network (home of Outstanding Leaders Partnership)</a></li>
         <li><a href="https://www.cefel.org.uk/npqh/">Church of England</a></li>
-        <li><a href="https://www.educationdevelopmenttrust.com/our-expertise/our-work-in-the-uk/our-programmes-in-the-uk/national-professional-qualifications-(1)/overview-of-our-npqs">Education Development Trust</a></li>
+        <li><a href="https://www.edt.org/united-kingdom/programmes/national-professional-qualifications-npqs/">Education Development Trust</a></li>
```
Comment (dalcJ) [inline app/views/headship.html:161]:
```suggestion
        <li><a href="https://www.edt.org/united-kingdom/discover-our-npqs/headship-npqh/">Education Development Trust</a></li>
```

File: app/views/leading-behaviour-and-culture.html [hunk 4]
```diff
@@ -155,7 +155,7 @@ <h2 class="govuk-heading-l">Providers</h2>
         <li><a href="https://www.ambition.org.uk/programmes/npqlbc/">Ambition Institute</a></li>
         <li><a href="https://www.outstandingleaders.org/NPQLBC">Best Practice Network (home of Outstanding Leaders Partnership)</a></li>
         <li><a href="https://www.cefel.org.uk/npqlbc/">Church of England</a></li>
-        <li><a href="https://www.educationdevelopmenttrust.com/our-expertise/our-work-in-the-uk/our-programmes-in-the-uk/national-professional-qualifications-(1)/overview-of-our-npqs">Education Development Trust</a></li>
+        <li><a href="https://www.edt.org/united-kingdom/programmes/national-professional-qualifications-npqs/">Education Development Trust</a></li>
```
Comment (dalcJ) [inline app/views/leading-behaviour-and-culture.html:158]:
```suggestion
        <li><a href="https://www.edt.org/united-kingdom/discover-our-npqs/leading-behaviour-and-culture-npqlbc/">Education Development Trust</a></li>
```

File: app/views/leading-literacy.html [hunk 5]
```diff
@@ -151,7 +151,7 @@ <h2 class="govuk-heading-l">Providers</h2>
 
       <ul class="govuk-list govuk-list--bullet">
         <li><a href="https://www.ambition.org.uk/programmes/npqll/">Ambition Institute</a></li>
-        <li><a href="https://www.educationdevelopmenttrust.com/our-expertise/our-work-in-the-uk/our-programmes-in-the-uk/national-professional-qualifications-(1)/overview-of-our-npqs">Education Development Trust</a></li>
+        <li><a href="https://www.edt.org/united-kingdom/programmes/national-professional-qualifications-npqs/">Education Development Trust</a></li>
```
Comment (dalcJ) [inline app/views/leading-literacy.html:154]:
```suggestion
        <li><a href="https://www.edt.org/united-kingdom/discover-our-npqs/leading-literacy-npqll/">Education Development Trust</a></li>
```

File: app/views/leading-teacher-development.html [hunk 6]
```diff
@@ -143,7 +143,7 @@ <h2 class="govuk-heading-l">Providers</h2>
         <li><a href="https://www.ambition.org.uk/programmes/npqltd/">Ambition Institute</a></li>
         <li><a href="https://www.outstandingleaders.org/NPQLTD">Best Practice Network (home of Outstanding Leaders Partnership)</a></li>
         <li><a href="https://www.cefel.org.uk/npqltd/">Church of England</a></li>
-        <li><a href="https://www.educationdevelopmenttrust.com/our-expertise/our-work-in-the-uk/our-programmes-in-the-uk/national-professional-qualifications-(1)/overview-of-our-npqs">Education Development Trust</a></li>
+        <li><a href="https://www.edt.org/united-kingdom/programmes/national-professional-qualifications-npqs/">Education Development Trust</a></li>
```
Comment (dalcJ) [inline app/views/leading-teacher-development.html:146]:
```suggestion
        <li><a href="https://www.edt.org/united-kingdom/discover-our-npqs/leading-teacher-development-npqltd/">Education Development Trust</a></li>
```

File: app/views/senior-leadership.html [hunk 7]
```diff
@@ -159,7 +159,7 @@ <h2 class="govuk-heading-l">Providers</h2>
         <li><a href="https://www.ambition.org.uk/programmes/npqsl/">Ambition Institute</a></li>
         <li><a href="https://www.outstandingleaders.org/npqsl">Best Practice Network (home of Outstanding Leaders Partnership)</a></li>
         <li><a href="https://www.cefel.org.uk/npqsl/">Church of England</a></li>
-        <li><a href="https://www.educationdevelopmenttrust.com/our-expertise/our-work-in-the-uk/our-programmes-in-the-uk/national-professional-qualifications-(1)/overview-of-our-npqs">Education Development Trust</a></li>
+        <li><a href="https://www.edt.org/united-kingdom/programmes/national-professional-qualifications-npqs/">Education Development Trust</a></li>
```
Comment (dalcJ) [inline app/views/senior-leadership.html:162]:
```suggestion
        <li><a href="https://www.edt.org/united-kingdom/discover-our-npqs/leading-teaching-npqlt/">Education Development Trust</a></li>
```

**Review by dalcJ — COMMENTED** [review]
Hi @SabrinaDfE
Should the links go to the specific NPQ pages on their site? It looks like that is what use for the other providers. I've added suggested links in, but have a check over

**Review by EmmaFrith — APPROVED** [review]

**Pull request merged**